### PR TITLE
Added logos of sponsors.

### PR DESCRIPTION
**Pull request by bravegnu**

### CODE REVIEW

**Comment by astronomersiva** [comment]
@bravegnu Will it be possible to get SVGs?

File: img/claylabs-logo.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (astronomersiva) [inline img/claylabs-logo.svg:1]: It's generally a good idea to run SVGs through https://jakearchibald.github.io/svgomg/ to trim file size.

**Review by astronomersiva — APPROVED** [review]

**Comment by bravegnu** [comment]
This is pretty much what they have, for now. I will check with the sponsors again.

File: img/claylabs-logo.svg (hunk 1 above)
Comment (bravegnu) [reply img/claylabs-logo.svg:1]: Thanks for the tip. Is there a command line tool to do that?

Comment (astronomersiva) [reply img/claylabs-logo.svg:1]: https://github.com/svg/svgo

But I prefer the web app as sometimes, it over-cleans and removes vital content too. The web app provides a preview after the cleanup so it's easy to catch if something goes wrong.

**Comment by astronomersiva** [comment]
@bravegnu , what tiers do these sponsors belong to?

**Comment by astronomersiva** [comment]
screen shot 2018-05-04 at 8 51 45 am

Something like this is ok? Gotta leave for office now. Will make changes tonight (Y)

**Comment by shrayasr** [comment]
Yes this works 👍 We can start here.

**Pull request merged**